static/app/views/ddm/queryBuilder.tsx

```diff
@@ -233,6 +242,8 @@ function MetricSearchBar({tags, mri, disabled, onChange, query}: MetricSearchBar
       placeholder={t('Filter by tags')}
       query={query}
       savedSearchType={SavedSearchType.METRIC}
+      projectIds={projectIds}
```

Comment:
i see the searchbar wants them as number, maybe lets convert them to number as close as possible to the searchbar

Reply:
ugh
https://github.com/getsentry/sentry/blob/master/static/app/components/events/searchBar.tsx#L160